app/src/main/AndroidManifest.xml

```diff
@@ -23,13 +23,17 @@
                 <category android:name="android.intent.category.LAUNCHER"/>
             </intent-filter>
         </activity>
-        <activity android:name=".ChoosingFolderActivity">
+        <activity android:name=".ChoosingFolderActivity"
+                  android:screenOrientation="portrait">
         </activity>
-        <activity android:name=".ExplainingChosenFolderActivity">
+        <activity android:name=".ExplainingChosenFolderActivity"
+                  android:screenOrientation="portrait">
         </activity>
-        <activity android:name=".PictureActivity">
+        <activity android:name=".PictureActivity"
+                  android:screenOrientation="portrait">
```

Comment:
make it sensorPortrait so that it would turn upside down when you turn up the device